helpers/safeTypeConvert.js

```diff
@@ -1,11 +1,14 @@
 const Handlebars = require("handlebars");
 const typeConvert = require("./typeConvert");
 
-const safeTypeConvert = (prop, shouldBox = false) => {
+const safeTypeConvert = (prop, shouldBox = false, inDeclaration = false) => {
```

Comment:
I'm not sure what you mean by a declaration here?

Reply:
Can you clarify when the annotation is and isn't added?

Reply:
> I'm not sure what you mean by a declaration here?

I should have called it `inFnSignature`. Because the annonations are only added on function declarations (i.e. next to the function signature.

> Can you clarify when the annotation is and isn't added?

This variant of the safeTypeConvert is only called on function definitions. In other places such as variable declarations in the method body [like below](https://github.com/ScottLogic/openapi-forge-java/blob/main/template/ApiClient.java.handlebars#L85)

```java
 {{safeTypeConvert _response.schema true}} responseObject;
```

the second optional parameter is not included, this defaults the behaviour to not add annotations. 

I'll add a comment as well to clarify.

Reply:
I changed the name and added a comment and applied your other feedback as well 👍